### ISSUE TITLE
Where are the Rust communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/rust#540]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Rust supported by one or more large organizations?
* Does Rust have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Rust? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Rust? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Rust? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Rust?
* Are there popular newsletters for Rust?
* Is Rust taught at programming bootcamps? (If so, what are some examples?)
* Is Rust taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Rust?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by bkhl** [comment]
Hope this helps:

* Is Rust supported by one or more large organizations?

It's sponsored [by Mozilla](https://research.mozilla.org/rust/).

* Does Rust have an official community manager?

Yes, there is a [community team](https://www.rust-lang.org/en-US/team.html), led by @ashleygwilliams

* Do you know of specific communities (online or offline) that are enthusiastic about Rust? (Chat communities, forums, meetups, student clubs, etc)

IRC channels on irc.mozilla.org, primarily #rust and #rust-beginners

[Rust User Forum](https://users.rust-lang.org/)

These are generally pleasant forums, thanks to a strong [code of conduct](https://www.rust-lang.org/en-US/conduct.html).

* Are there popular conferences for Rust? (If so, what are some examples?)

[RustFest](https://paris.rustfest.eu/) in Europe.

[Rust Belt Rust](http://www.rust-belt-rust.com/) in the US.

* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Rust? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

Yes, [RustBridge](https://rustbridge.github.io/)

* Are there specific groups or programs dedicated to mentoring people in Rust?

Yes, issues with mentorship available can be found through https://www.rustaceans.org/findwork/starters

* Are there popular newsletters for Rust?

[This Week in Rust](https://this-week-in-rust.org/)

* Is Rust taught at programming bootcamps? (If so, what are some examples?)

Don't know.

* Is Rust taught at universities? (If so, what are some examples?)

[Example of complete courses at University of Virginia](http://rust-class.org/)

[Part of language survey course at University of Maryland](https://www.cs.umd.edu/class/spring2018/cmsc330/)

**Comment by kytrinyx** [comment]
❤️ 💛 💚 💙 💜 🖤 @steveklabnik this is awesome thanks. 

> It really depends on what you mean exactly by "community manager" though; I've found that that role's responsibility varies a lot between organizations. Happy to elaborate more if you want to dig into this question more.

I might ping you to dig more into this. For now I'm trying to get an overview of what things look like in general across the board.

> It also depends on what you mean by "support"; do you mean funding the language itself, or something else? 

The question was deliberately left ambiguous, as I'm curious what sort of support is available. This information is ✨

**Comment by coriolinus** [comment]
@kytrinyx Do you have the information you need from this issue? If yes, I'm inclined to close it now.

**Comment by kytrinyx** [comment]
I do, thank you!
